native-to-erc20/oracle/.env.example

```diff
@@ -41,3 +41,8 @@ REDIS_LOCK_TTL=1000
 
 LOG_LEVEL=debug
 MAX_PROCESSING_TIME=20000
+
+ODDIN_SCORE_URL=https://o-api.oddin.ai/fuse/wallet/validity
```

Comment:
Ilan are you prepare for DDoS? We might want to hide this from the public config